name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/security/SecurityPerformanceRecord.java

```diff
@@ -594,7 +601,8 @@ private void calculateDividends(CurrencyConverter converter)
         this.sumOfDividends = dividends.getSum();
         this.dividendEventCount = dividends.getNumOfEvents();
         this.lastDividendPayment = dividends.getLastDividendPayment();
-        this.rateOfReturnPerYear = dividends.getRateOfReturnPerYear();
+        this.dividendRateOfReturnPerYear = dividends.getRateOfReturnPerYear();
+        dividendYieldOnCost = dividends.getYieldOnCost();
```

Comment:
Using `this` would be more consistent.

Reply:
I think this it's a worthless change but if it makes you happy... I add it now.